src/darker/tests/test_git.py

```diff
@@ -143,16 +143,17 @@ def test_get_missing_at_revision_worktree(git_repo):
 
 
 def test_git_diff_name_only(git_repo):
-    """``_git_diff_name_only()`` only returns paths of modified files"""
+    """``_git_diff_name_only()`` only returns paths of modified files and ignores
+    renamed/moved files"""
     git_repo.add({"a.py": "a", "b.py": "b", "c.py": "c"}, commit="Initial commit")
     first = git_repo.get_hash()
     git_repo.add({"a.py": "A", "b.dy": "B"}, commit="only a.py modified")
+    git_repo.rename("c.py", "x.py", commit="rename c.py to x.py")
     second = git_repo.get_hash()
 
     result = git._git_diff_name_only(
         first, second, {Path("a.py"), Path("c.py"), Path("Z.py")}, git_repo.root
     )
```

Comment:
In test functions, I've followed the convention of separating the invocation of tested code with single blank lines above and below. This helps visually recognize the setup, test and verification parts of the test function.
```suggestion
    )

```